packages/create-react-app/index.js

```diff
@@ -136,6 +136,16 @@ function createApp(name, verbose, version) {
   run(root, appName, version, verbose, originalDirectory);
 }
 
+function createFolderPath(path) {
+  var projectPath = (path + "/");
```

Comment:
I'm in windows. with the slash it's working fine. 

Are you talking about the backslash?